### DISCUSSION TITLE
官方能提供rocketmq5.0 metrics 的grafana的dashboards 模板吗

### DISCUSSION THREAD

**Discussion by letian-tang** [apache/rocketmq#7212] [Q&A]
目前 https://grafana.com/grafana/dashboards/
都是4.0的，不适合5.0
自己弄个又有学习的成本，期待官方放出来一个模版

**Comment by joeCarf** [comment]
4.x的大盘地址能发一下吗，我参考下能否贡献 :)

**Reply by zhiliatom** [reply]
hi，look at there `https://github.com/apache/rocketmq-exporter/blob/master/rocketmq_exporter_overview.json`
